components/BannerNotification.tsx

```diff
@@ -7,13 +7,19 @@ import testIDProps from '../shared/commonUtil';
 
 export const BannerNotification: React.FC<BannerNotificationProps> = props => {
   return (
-    <View {...testIDProps(props.testId)}>
-      <Row style={Theme.Styles.downloadingVcPopUp}>
+    <View {...testIDProps(props.testId)} style={props.customStyle}>
+      <Row
+        style={
+          props.customStyle
+            ? Theme.Styles
+                .backupRestoreBanner /* TODO: rename to style without mentioning any specific flow */
```

Comment:
Pending todo?

Reply:
Yes, since this is a common component, we were thinking if we could rename the styles without mentioning any flow like backup, download vc, etc.. 

Since this component is getting used in other places as well, we thought this could take up extra time to refactor and check. For this reason we added TODO for the moment
cc: Pooja